.github/workflows/php.yml

```diff
@@ -12,8 +12,8 @@ jobs:
     strategy:
       matrix:
         os: ['ubuntu-latest']
-        php: ['8.0', '8.1', '8.2']
-    continue-on-error: ${{ matrix.php == '8.2' }}
+        php: ['8.0', '8.1', '8.2', '8.3', '8.4']
```

Comment:
after 8.0 is EOL - maybe take it out completely?

Reply:
Yeah right, we drop it not...